build.gradle

```diff
@@ -30,7 +24,8 @@ def jettyVersion = '9.3.8.v20160314'
 def jerseyVersion = '2.22.2'
 
 dependencies {
-    compile 'org.terasology.engine:engine:1.0.0'
+    //compile 'org.terasology.engine:engine:1.0.0'
+    compile project(':engine')
```

Comment:
Does this use the local engine project instead of the artefact? If so, is this just for (local) testing or should it stay in?